packages/utilities/psammead-storybook-helpers/src/index.stories.jsx

```diff
@@ -35,4 +35,8 @@ storiesOf('Input Provider', module)
       ),
     ),
     { notes, knobs: { escapeHTML: false } },
-  );
+  )
+  .addDecorator(dirDecorator)
```

Comment:
![image](https://user-images.githubusercontent.com/11341355/57363352-d67c5f00-7178-11e9-8d70-adb80aa36d90.png)

You may need to split these up, like I have [here](https://github.com/bbc/simorgh/blob/ab5ddf1f30dc01e575497b3369eb11178886449b/src/app/containers/StoryPromo/index.stories.jsx#L20-L26)